src/pages/open-source.js

```diff
@@ -4,14 +4,13 @@ import { StaticQuery, graphql } from 'gatsby'
 import Head from '../components/Common/Head'
 import Layout from '../components/layout'
 import BlueBackground from '../components/Common/BlueBackground'
-import { Grid } from '../components/grid'
 import CaseStudyPreview from '../components/Common/CaseStudyCards/CaseStudyPreview'
 import Statement from '../components/Common/Statement'
 
+import TalksSection from '../components/OpenSource/Talks'
 import PartnershipsSection from '../components/OpenSource/Partnerships'
 
 import Contributions from '../components/OpenSource/Contributions'
-import OpenDeliverables from '../components/OpenSource/OpenDeliverables'
```

Comment:
this as well

Reply:
Yep, related to the above which I probably removed by mistake.